ingenialink/canopen/network.py

```diff
@@ -391,8 +389,8 @@ def _setup_connection(self) -> None:
         """
         if self._connection is None:
             self._connection = canopen.Network()
-            if self.__device == CAN_DEVICE.IXXAT.value:
-                self._connection.listeners.append(CustomIXXATListener())
+            if self.__device in [CAN_DEVICE.IXXAT.value, CAN_DEVICE.KVASER.value]:
```

Comment:
Any context why __device is a string instead of the enum?

Reply:
I'm not sure, maybe it's for backward compatibility?

Reply:
It's a private attribute, the compatibility should not be an issue